### PR TITLE
Sync upstream stable/victoria

### PR DESCRIPTION
**Pull request by cityofships**

### CODE REVIEW

**Comment by cityofships** [comment]
Waiting for https://review.opendev.org/c/openstack/kolla-ansible/+/782735 to be merged

File: ansible/group_vars/all.yml [hunk 1]
```diff
@@ -668,7 +669,6 @@ enable_neutron_port_forwarding: "no"
 enable_nova_serialconsole_proxy: "no"
 enable_nova_ssh: "yes"
 enable_octavia: "no"
-enable_octavia_driver_agent: "{{ enable_octavia | bool and neutron_plugin_agent == 'ovn' }}"
```
Comment (markgoddard) [inline ansible/group_vars/all.yml:671]: I think this came from a backport?

Comment (cityofships) [reply ansible/group_vars/all.yml:671]: Backported from a patch in progress in #151, which also caused a merge conflict later. I propose a revert, synchronisation with the upstream and backporting https://review.opendev.org/c/openstack/kolla-ansible/+/783699 again

**Comment by bbezak** [comment]
ready for review - after https://review.opendev.org/c/openstack/kolla-ansible/+/782735  got merged, and readded - https://github.com/stackhpc/kolla-ansible/pull/158/commits/0a23519cdc2bb624a9bb051ac544243b78451349

**Review by jovial — APPROVED** [review]
Looks sane. I don't see any conflicts marked in the commit message, so I assume it was a clean merge/cherry-pick.

**Comment by bbezak** [comment]
> Looks sane. I don't see any conflicts marked in the commit message, so I assume it was a clean merge/cherry-pick.

Actually I needed to re-add https://github.com/stackhpc/kolla-ansible/pull/158/commits/90bbd2b1f5bff918f40b4c7d7cbcb24b742f27ef because of conflict between https://github.com/stackhpc/kolla-ansible/pull/158/commits/326e15d580bc021bf3bc3370f00612f848f4a7c9 and backport of https://review.opendev.org/c/openstack/kolla-ansible/+/783699

**Review by markgoddard — APPROVED** [review]

**Pull request merged**